main.rb

```diff
@@ -31,7 +34,11 @@ def start
       when 1 then @book_handler.list_books
       when 2 then @book_handler.list_labels
       when 3 then @book_handler.add_book
-      when 10 then @book_handler.quit
+      when 4 then @app.list_all_music_albums
+      when 5 then @app.list_all_genres
+      when 6 then @app.create_a_music_album
+      when 10 then @app.create_a_genre
+      when 11 then @book_handler.quit
       else
         puts 'Invalid option. Please try again.'
       end
```

Comment:
suggestion kindly seperate the define start method into two method it will work for the linters to work there so many lines error we can define two method